vendor/scriptJs.js

```diff
@@ -0,0 +1,87 @@
+
+jQuery(function() {
+	"use strict";
+
+	var Com = Com || {};
+	var $FEEDFW = Com; 
+
+	$FEEDFW.show = {
+		// Use create element method which I have used through my current work place 
+		createElement: function(id, targetDiv, type, className) {
+			var elm = document.createElement((type) ? type : 'div');
+			if(className) {
+				elm.className = className;
+			}
+			if(id){
+				elm.id = id;
+			}
+			if(targetDiv) {
+				targetDiv.appendChild(elm);
+			}
+			return elm;
+		},
+
+	};
+
+	$FEEDFW.action = {
```

Comment:
"action" is a bit generic of a name...how would you describe what this method is doing?